TopAnalysis/interface/FCNCTriLeptonCppWorker.h

```diff
@@ -22,7 +22,7 @@ class FCNCTriLeptonCppWorker {
   ~FCNCTriLeptonCppWorker() = default;
 
   void setMuons(TRAF pt, TRAF eta, TRAF phi, TRAF mass, TRAI charge,
-                TRAF relIso, TRAB isTight, TRAB isGlobal, TRAB isPFcand, TRAB isTracker);
+                TRAF relIso, TRAB isLoose, TRAB isTight, TRAB isGlobal, TRAB isPFcand, TRAB isTracker);
```

Comment:
is there Muons_isLoose in the NanoAOD? you have to use isGlobal, isPFcan and isTracker to define isLoose